package-lock.json

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
@ryangraham-ge Hey, check out https://goodeggs.atlassian.net/wiki/spaces/ENG/pages/459833453/Using+NPM+and+Yarn. We use the `yarn` CLI exclusively at Good Eggs. Since they aren't compatible, you should make sure never to run `npm` or `npx`! Please delete this file!